edxval/tests/test_api.py

```diff
@@ -436,3 +530,90 @@ def test_get_info_queries_for_only_video(self):
         with self.assertNumQueries(4):
             api.get_video_info(constants.VIDEO_DICT_ZEBRA.get("edx_video_id"))
 
+@ddt
+class TestRerunCourse(TestCase):
+    """Tests rerun course"""
+
+    def setUp(self):
+        """
+        Creates a course for testing
+
+        Creates two videos, each with two encodings.
```

Comment:
This should also create a video not in the test course to make sure that only videos in the test course get copied